cmd/create/cluster/cmd.go

```diff
@@ -735,6 +739,13 @@ The password must
 			"associated with intended shared VPC, e.g., '1vo8.p1.openshiftapps.com'.",
 	)
 
+	flags.StringSliceVar(
+		&args.additionalComputeSecurityGroupIds,
+		additionalComputeSecurityGroupIdsFlag,
+		nil,
```

Comment:
Would it be better to avoid nil and use an empty slice?

Reply:
Nil helps in identifying if the value changed in some cases where we can't use cmd.Flags for some reason, that seems to be a standard looking at other `StringSliceVar`. We may want to revisit that later to adjust so all aren't null